notebooks/fastText_viz.ipynb

```diff
@@ -8,17 +8,37 @@
     "In this notebook I train an unsupervised model and view the word vectors that fastText comes up with. I do a little bit of exploration of these word vectors. These methods can be used for further visualization of other trained models for transparency with the model."
```

Comment:
<p>Could you please also fix this output :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/github-labeler/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>